### ISSUE TITLE
Get rid of "stories" everywhere

### ISSUE THREAD

**Issue by ralsina** [getnikola/nikola#1891]
The "stories" term is a remnant of my own config before I even started writing Nikola. In fact, the term comes from PyDS, a blogging software so old it doesn't even have a website anymore.

So, the right term is "pages", let's use it.
- [x] Remove "stories" from the docs
- [x] Remove "stories" from the config (for new sites at least)
- [x] Remove "stories" from the code (if it's still there) (done except for public APIs)
- [x] Move `story.tmpl` code to `page.tmpl`, leaving `story.tmpl` as a stub

Once we are using pages consistently everywhere, this closes.

**Comment by tritium21** [comment]
How would that last point affect users who have a custom template named page.tmpl (which inherits from story.tmpl)?

**Comment by christianp** [comment]
Hi, I found this issue through up-for-grabs.net. I've done the easy part of this, replacing all instances of 'story' and 'stories' with 'page' and 'pages', in christianp/nikola@2dfe792d93440c5c1a203df328a6641bc58a0e2f.
I renamed two config settings: `STORY_INDEX` and `COMMENTS_IN_STORIES`, which will cause upgrade headaches. And the point about `page.tmpl` potentially already existing is a good one.
